README.md

```diff
@@ -166,6 +166,14 @@ default: `0`
 
 Use this to configure how long it should take before the `QRCodeScanner` should reactivate.
 
+
+#### `cameraTimeout`
+propType: `number`
+default: `0`
+
+Use this to configure how long it should take before the `QRCodeScanner` should be displayed. After that the camera will be inactive and press the view to reactivate it.
+The `0` default means it's always on.
```

Comment:
maybe would be great to describe that the number expected in this property must to be in milliseconds, what do you think?

Reply:
Yes that would be good. I'll resubmit the PR